manifests/d/Discord/Discord/PTB/1.0.1129/Discord.Discord.PTB.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Discord.Discord.PTB
+PackageVersion: 1.0.1129
+InstallerType: exe
+Scope: user
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install

```

Comment:
Should this be changed to `deny` and `RequireExplicitUpgrade` to match `Discord.Discord` ?